sdk/pom.xml

```diff
@@ -91,11 +106,11 @@
               <outputTargets>
                 <outputTarget>
                   <type>java</type>
-                  <outputDirectory>${project.build.sourceDirectory}</outputDirectory>
+                  <outputDirectory>${protobuf.output.directory}</outputDirectory>
                 </outputTarget>
                 <outputTarget>
                   <type>grpc-java</type>
-                  <outputDirectory>${project.build.sourceDirectory}</outputDirectory>
+                  <outputDirectory>${protobuf.output.directory}</outputDirectory>
```

Comment:
I am moving auto-generated code to the auto-generated folder and out of source control. This way, we only need to version the proto files.